commerce_coordinator/apps/rollout/pipeline.py

```diff
@@ -51,11 +59,42 @@ def run_filter(self, request):
         elif sku:
             active_order_management_system = FRONTEND_APP_PAYMENT_CHECKOUT
         else:
-            logger.exception(f'An error occurred while determining the active order management system.'
-                             f'No waffle flag, course_run_key or sku value found')
-            raise OpenEdxFilterException(f'Neither course_run_key and waffle flag value nor sku found.'
-                                         f'Unable to determine active order management system.')
+            logger.exception('An error occurred while determining the active order management system.'
+                             'No waffle flag, course_run_key or sku value found')
+            raise OpenEdxFilterException('Neither course_run_key and waffle flag value nor sku found.'
+                                         'Unable to determine active order management system.')
 
         return {
-            'active_order_management_system': active_order_management_system
+            ACTIVE_ORDER_MANAGEMENT_SYSTEM_KEY: active_order_management_system
         }
+
+
+class DetermineActiveOrderManagementSystemByOrder(PipelineStep):
+    """ Using an Order Numer/ID to determine the active order management system """
+
+    def run_filter(self, order_number, **kwargs):  # pylint: disable=arguments-differ
+        """ Using an Order Numer/ID to determine the active order management system """
+
+        active_order_management_system = None
+
+        if is_uuid(order_number):
+            active_order_management_system = COMMERCETOOLS_ORDER_MANAGEMENT_SYSTEM
+        elif is_legacy_order(order_number):
+            active_order_management_system = ECOMMERCE_ORDER_MANAGEMENT_SYSTEM
```

Comment:
Can we only check for `is_legacy_order()` and fallback to CT order always? The reason is that we know our legacy order number starts with `EDX-...`. Also, in the future, we plan to move away from CT Order's UUID and will be using human-readable order_number.

Reply:
I was thinking we may have other systems in the future... so i was being thorough... but sure.